src/melee/ft/chara/ftSeak/ftseak.c

```diff
@@ -62,3 +62,12 @@ void ftSeak_OnKnockbackEnter(HSD_GObj* fighterObj) {
 void ftSeak_OnKnockbackExit(HSD_GObj* fighterObj) {
     Fighter_OnKnockbackExit(fighterObj, 1);
 }
+
+
+// 8011412C - 80114160
+// https://decomp.me/scratch/b1oIZ
+// void lbl_8011412C(HSD_GObj* fighterObj) {
+//     Fighter* fighter = fighterObj->user_data;
+//     fighter->cb.x21BC_callback_Accessory4 = 0;
+//     func_8007EFC8(fighterObj, &ftZelda_8013B4D8);
+// }
```

Comment:
Can be added as an `extern` placeholder until Sheik is matched.
```suggestion
// 8011412C - 80114160
// https://decomp.me/scratch/b1oIZ
extern void lbl_8011412C(HSD_GObj* fighterObj);
// void lbl_8011412C(HSD_GObj* fighterObj) {
//     Fighter* fighter = fighterObj->user_data;
//     fighter->cb.x21BC_callback_Accessory4 = 0;
//     func_8007EFC8(fighterObj, &ftZelda_8013B4D8);
// }
```